Shared_Il2cpp/ClothingStateMenuPlugin.cs

```diff
@@ -34,7 +34,12 @@ public class ClothingStateMenuPlugin : BasePlugin
         private static readonly List<GUIContent[][]> _AccessoryButtonContentCache = new();
 
         private readonly List<IStateToggleButton> _buttons = new();
+#if SVS
         private const int CoordCount = 3; //todo make this not hardcoded
+#elif AC
+        // IDHI: maybe this way
+        private static readonly int CoordCount = Enum.GetValues(typeof(ChaFileDefine.CoordinateType)).Length;
+#endif
```

Comment:
Ideally this would be checked dynamically to proactively support any plugins that add additional coordinate slots. Actually I believe there is one for SVS already. [Here's how it's done in the KK branch](https://github.com/ManlyMarco/Illusion_ClothingStateMenu/blob/a83f7f5f3e2ed74d1bb00de97bf74917ca24f6c9/Shared_Koi/ClothingStateMenuPlugin.cs#L278). This is not a priority however.